source/_integrations/homepilot.markdown

```diff
@@ -0,0 +1,28 @@
+---
+title: "Rademacher HomePilot"
+description: "Instructions for connecting Rademacher HomePilot devices."
+ha_release: "2021.5"
+ha_category: Cover
+ha_iot_class: "Local Polling"
+ha_config_flow: true
+ha_codeowners:
+  - '@nico0302'
+ha_domain: homepilot
+---
+
+The [Rademacher HomePilot](https://www.rademacher.de/smart-home/produkte/homepilot) integration allows you to connect your Rademacher HomePilot Hub or Smart2Smart Bridge with Home Assistant.
+The [pyhomepilot library](https://github.com/Nico0302/pyhomepilot) is used for communicating.
```

Comment:
The underlying does not add anything for the end-user and should not be part of the user documentation.

```suggestion
```